flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/DeduplicateITCase.scala

```diff
@@ -226,6 +226,42 @@ class DeduplicateITCase(miniBatch: MiniBatchMode, mode: StateBackendMode)
     assertEquals(expected.sorted, rawResult.sorted)
   }
 
+  @Test
+  def testLastRowOnRowTimeFollowedByUnboundedAgg(): Unit = {
```

Comment:
we should enable  `minibatch` here ? and we can also add a test for firstRow

Reply:
@leonardBang Thanks for suggestion. DeduplicateITCase already tests four types of configuration for each case:
MiniBatchOn + Heap, MiniBatchOn + RocksDB, MiniBatchOff + Heap, MiniBatchOff + RocksDB, so I don't configure miniBatch in this case. 
I could add a test for FirstRow.

Reply:
hah, you're wright @beyond1920

Reply:
yes. DeduplicateTest is to test plan is same as expected when `Deduplicate` is followed by unbounded agg, DeduplicateITCase is to test result is same as expected in this case.

Reply:
Okay, I'm ok with this change, that's better if you could add a test for lastRow.